src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -13,6 +13,16 @@
 
 import codeu.model.data.User;
 import codeu.model.store.basic.UserStore;
+import java.util.List;
+
+import com.google.cloud.translate.Detection;
```

Comment:
Are these import statements used anymore?

Reply:
Do not think so. Sorry about that, had to do a LOT of refactoring to get it to work how I wanted